automatic/spotify/update.ps1

```diff
@@ -0,0 +1,43 @@
+Import-Module AU
+
+# We only use this for checking for Spotify versions, nothing else
+$releasesUrl = 'http://www.filehorse.com/download-spotify/'
+$downloadUrl = 'https://download.spotify.com/SpotifyFullSetup.exe'
+
+function global:au_BeforeUpdate {
+  # we need to verify that the downloaded file is the correct version
+  Get-RemoteFiles -Purge
+  $file = Get-ChildItem "$PSScriptRoot\tools" -Filter "*.exe" | select -First 1 | % { Get-Item $_.FullName }
```

Comment:
You can do it that way I guess ...
Don't know whats frong with `gi tools\*.exe | select -first 1`